internal/cli/atlas/deployments/scripts/start_mongot.sh

```diff
@@ -14,11 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if [ ! -f "$KEYFILEPATH" ]
```

Comment:
without this check, the restarting a container was failing, is this not the case anymore?

Reply:
not the case, I've tested it

Reply:
also @tibulca I think we should make a test for this, it seems an important enough regression for us to test